### DISCUSSION TITLE
Your application cannot be reviewed until you fix the following:

### DISCUSSION THREAD

**Discussion by Vargrak** [community/community#37159] [GitHub Education] [answered]
I cannot get past the pre-check for the student developer pack. I've contact support and they closed it in minutes with a link to an FAQ that was to lack of documentation.

I have a valid school email and I have a valid school ID that I've tried to submit on multiple occasions now, but I keep running into this error:

> Your application cannot be reviewed until you fix the following:
`Your school is a GitHub Campus partner. Please contact your school administrator to have the correct benefits applied to your GitHub account.

Modify your browser settings to allow GitHub to use your location information. Once location is allowed you MUST cancel your current application and begin again.

You are significantly more likely to be verified if you have completed your GitHub user profile with your full name and a short bio.`

I do not know who my school administrator would be. There are multiple departments this would be useful too. Nowhere on their website externally or internally has any information. My department has not given me any information on this. My department's discord simply links to the benefits page with no details of who I would contact.

I have changed the browser location settings before and received the same error. Other people reporting this issue have not had any responses or their issue resolved. My ticket is still open.

**Comment by imqcro77** [comment]
Have you fixed this?

**Reply by Vargrak** [reply]
No luck so far.
I haven't heard back from GitHub support yet. I also asked my department's IT support and he seems just as confused. He said I should just be able to use my email and it would work. He said he would try to replicate my problem and get back to me. But that was about a week ago.

**Reply by ascendho** [reply]
This problem occurred to me too, maybe you should try to email GitHub Support via school-issued email with your valid studying certificate attachment, hope they will manually accept your application.

**Reply by imqcro77** [reply]
> This problem occurred to me too, maybe you should try to email GitHub Support via school-issued email with your valid studying certificate attachment, hope they will manually accept your application.

I reapplied from my phone and it worked.

**Reply by ascendho** [reply]
Oh, that’s great!

**Reply by Vargrak** [reply]
Didn't work on my phone unfortunately. What's github's support email? I tried looking it up but most of their contact information just goes to the ticket system.

**Reply by ascendho** [reply]
> Didn't work on my phone unfortunately. What's github's support email? I tried looking it up but most of their contact information just goes to the ticket system.

Try this, edu-noreply@github.com.

**Reply by samus-aran** [reply]
Hi @ascendho, that email is unmanned so we recommend not using that one.

@Vargrak :wave: I'm sorry you are having trouble gaining verification. [Updated] Looking into it further Partner schools can use the Global Campus application form to verify their students. Your school email domain should be adequate. Another way Partner schools can verify is using their own portal/URL which is what I was referring to in my earlier edit. So apologies on the misunderstanding.

**Reply by ascendho** [reply]
> > Didn't work on my phone unfortunately. What's github's support email? I tried looking it up but most of their contact information just goes to the ticket system.
> 
> 
> 
> Try this, edu-noreply@github.com.

Do you apply it on phone? You can switch to computer, it may work successfully, but you may need to print the studying certificate first so that it can be captured by webcam.

**Reply by DeepuKr0315** [reply]
@samus-aran i tried but it didn't work as well

**Reply by arkanide1** [reply]
i have the same issue

**Reply by Umirek** [reply]
I got a really generall error. Tried on mobile, web, different computers, web browser... No fix. 
![image](https://github.com/user-attachments/assets/82ed8ff0-57bd-427d-9ab3-adfff0c5e0c8)

**Reply by 22125A3707** [reply]
![Screenshot 2025-04-15 145548](https://github.com/user-attachments/assets/91361f72-c821-4b2f-aa35-37c43e78a05b)
I am getting the same error i tried login logout changed browsers and all still having the same issue

**Comment by samus-aran** [comment, marked as answer]
@Vargrak Can you try and re-apply again (if you need help canceling the application click 'Reset'). Your academic email should be enough to verify you. Let us know how you get on or if you need guidance re-applying 👍

**Reply by Vargrak** [reply]
> @Vargrak Can you try and re-apply again (if you need help canceling the application click 'Reset'). Your academic email should be enough to verify you. Let us know how you get on or if you need guidance re-applying +1

Seems to have worked now. Thank you!

**Reply by samus-aran** [reply]
Awesome! 🎉

**Reply by Ratangulati** [reply]
> @Vargrak Can you try and re-apply again (if you need help canceling the application click 'Reset'). Your academic email should be enough to verify you. Let us know how you get on or if you need guidance re-applying 👍

How to cancel the application? Can you please help me with that?

**Reply by arnavmeduri** [reply]
I have the same question. How do you cancel your application? Can someone help walk through the steps please!

**Reply by EzCryp** [reply]
@Vargrak I am having the same issue. Trying to upload many times, What else can I do? Thanks!

**Reply by valcero** [reply]
what if college has not provided an email address. What then?

**Comment by K0I0S-Codes** [comment]
I'm having a similar issue but with the below message instead, and it may not show in my profile now, I did change my information to display my full name in my profile name, and even in my profile Readme, but nothing changes. (I've changed it back now, as I don't want my full name just all over the internet but I still have my first name and last name initial in my Readme)
![image_2022-11-11_141242447](https://user-images.githubusercontent.com/23627450/201413604-5f9086a8-07a9-4524-b080-59ad1004fd55.png)

**Reply by daniellemdavid** [reply]
Hi @SpyrossS3,
I had the same issue and using your post I was able to solve it. The message says to complete the user profile with information such as your name as "it appears in your image". This is were the solution for my problem was found. 

Possible Solution:
- Ensure that the .jpeg image you submit in your application displays your name.
    The first time I submitted it was a screenshot of my class schedule which did not include my full name (only my student username). I changed this to a screenshot of my unoffical transcript which did include my name.

- Ensure that your name is displayed on your user profile before submitting.
    I assume this has to match the name in the .jpeg in order for it to work. 

I will also add that, after clicking submit, the processing seemed to be taking forever (more than 5 mins had passed). I then went to another tab to login to education.github.com again and it seemed that my application had been approved while it was still "processing". 

Hope that helps and good luck,
ellechaotech

**Reply by K0I0S-Codes** [reply]
I appreciate it, but I've tried that several times already to no success. I'm glad your issue was resolved, though!

**Comment by rieog** [comment]
Your application cannot be reviewed until you fix the following:
Your school is a GitHub Campus partner. Please contact your school administrator to have the correct benefits applied to your GitHub account.


why am i getting this error can someone please help me

**Reply by ghost** [reply]
Ask your school admin about github education pack. It usually might be your School IT person. For me, it was my computer teacher

**Comment by yikafu** [comment]
I have submitted various documents on GitHub Education to prove that I am a student, including student ID card, third-party certification, school admission letter, etc., and  modified my personal information.I've tried emailing them, but they've now asked to contact them through the community.
I don't think there's anything wrong with my information, but I keep getting rejected automatically. I have tried various methods.The following is the reason for rejection I have been getting. I've been rejected 22 times。😭
However, I suddenly found the following tips on the "Get your Git Hub benefits" page. I don't know if it's relevant
why am i  reject can someone please help me
<img width="542" alt="屏幕截图 2023-03-11 130823 jpg" src="https://user-images.githubusercontent.com/98627904/224467061-6369c085-b3f5-4536-b65a-32e9b4ac8b06.png">
<img width="292" alt="image" src="https://user-images.githubusercontent.com/98627904/224466921-45f91272-90fe-4286-96cd-914586fb597d.png">

**Reply by arkanide1** [reply]
same problem man😭😭

**Reply by arkanide1** [reply]
Have you fixed this?

**Comment by zyq-20000901** [comment]
I have the same mistake.....

**Comment by FaGr-Devlab** [comment]
As Vargrak (author of this post) I have a valid school email and I have a valid school ID-card but I keep running into this error:
`Your school is a GitHub Campus partner. Please contact your school administrator to have the correct benefits applied to your GitHub account.'

I asked to IT department and they said that they already known the problem and that they have opened a ticket....and this was 2 weeks ago.

How I can resolve it ?

Thanks,
F.G.

**Comment by prravda** [comment]
TL;DR
Use smartphone and take a picture of printed-out certificate.

---

IDK how it works, I solved with this problem using my smartphone.

I applied this student pack using scanned certificate of current enrollment, but it was rejected.
So I changed my github profile. I changed my company to my university, my mail to my university's web mail. 
Then I tried again but it failed too.

So I found this thread and try to use smartphone and take a picture of printed certificate, not scanned one. 

Finally, I pass the processes. 

Thanks to @Vargrak

**Comment by taher182** [comment]
I'm getting this issue after uploading my id card

```
Your application cannot be reviewed until you fix the following:
Your uploaded proof has already been used previously.
```

**Reply by ghost** [reply]
Hello there! I'm facing the same issues. Did you find any fix?

**Reply by Priyanshu09T** [reply]
Didi you find any solution?

**Comment by taher182** [comment]
Not yetOn Dec 25, 2023 4:58 PM, SAYANDHRAJ ***@***.***> wrote:
Hello there! I'm facing the same issues. Did you find any fix?

**Comment by youyeg** [comment]
I have the same issue.

**Comment by 2leeje** [comment]
"Your application cannot be reviewed until you fix the following:
Your uploaded proof has already been used previously."

I got this issue so many times, and here is my solution!!
I cut some of the photos(in my case, enrollment document) I submitted, saved them as new files, and resubmitted them. 
And it passed!

hope it'll help you guys :)

**Reply by niku230** [reply]
Worked, thanks. Also as a hint, the submitted picture has to be below 500kb.

**Comment by sailakshman1** [comment]
Your application cannot be reviewed until you fix the following:

You have an outstanding discount request for @sailakshman1. We will get back to you soon.
